<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorparameter</class>
 <widget class="QDialog" name="colorparameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="validate_Button">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refuse_Button">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="redLabel">
           <property name="text">
            <string>Rouge</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="redSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="redSlider">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="redValueLabel">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="greenLabel">
           <property name="text">
            <string>Vert</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="greenSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>6</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="greenSlider">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="greenValueLabel">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="blueLabel">
           <property name="text">
            <string>Bleu</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="blueSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="blueSlider">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="blueValueLabel">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>refuse_Button</sender>
   <signal>clicked()</signal>
   <receiver>colorparameter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validate_Button</sender>
   <signal>clicked()</signal>
   <receiver>colorparameter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
